kafka/src/main/java/com/expedia/adaptivealerting/kafka/visualizer/AnomalyModel.java

```diff
@@ -0,0 +1,75 @@
+package com.expedia.adaptivealerting.kafka.visualizer;
+
+import com.expedia.adaptivealerting.anomdetect.detect.AnomalyThresholds;
+import com.expedia.metrics.TagCollection;
+
+public class AnomalyModel {
```

Comment:
Please use Lombok's `@Data` annotation here (as opposed to explicitly defining all the accessors), and please make the member fields private.